ush/ufsda/stage.py

```diff
@@ -106,26 +103,74 @@ def atm_obs(config):
         r2d2_config['target_file_fmt'] = target_file
         r2d2_config['obs_types'] = [ob['obs space']['name']]
         ufsda.r2d2.fetch(r2d2_config)
+
+def bias_obs(config):
+    # fetch bias files
+    r2d2_config = {
+        'start': config['prev_valid_time'],
+        'end': config['prev_valid_time'],
+        'step': config['atm_window_length'],
+        'dump': 'gdas',
+        'experiment': 'oper_gdas',  # change this here and other places to be oper_{dump}
+        'target_dir': config.get('target_dir', config.get('BKG_DIR', './')),
+    }
+    r2d2_config = NiceDict(r2d2_config)
+    # get list of obs to process and their output files
+    obs_list_yaml = config['OBS_LIST']
+    obs_list_config = Configuration(obs_list_yaml)
+    obs_list_config = ufsda.yamltools.iter_config(config, obs_list_config)
+    for ob in obs_list_config['observations']:
+        r2d2_config.pop('file_type', None)
+        r2d2_config['obs_types'] = [ob['obs space']['name']]
         # get bias files if needed
         if 'obs bias' in ob.keys():
             r2d2_config['type'] = 'bc'
             r2d2_config['provider'] = 'gsi'
             r2d2_config['start'] = config['prev_valid_time']
             r2d2_config['end'] = r2d2_config['start']
+
+            # fetch satbias
             r2d2_config['file_type'] = 'satbias'
             target_file = ob['obs bias']['input file']
             r2d2_config['target_file_fmt'] = target_file
-            ufsda.r2d2.fetch(r2d2_config)
+            print(f"config is {config}")
+            try:
```

Comment:
Instead of try, except, we can do:

```
r2d2_config['experiment'] = config.get('experiment', 'oper_gdas')
```

The .get method will return the key value if it exists, otherwise the default 'oper_gdas' is selected.

Reply:
Wasn't sure where to put this comment, but how does the workflow know when to send 'experiment' and when to not? Like the first analysis it should use `oper_gdas` I would think, for example.

Reply:
I manually prime parallels by copying into ROTDIR the files needed to start the experiment.  It looks like g-w `setup_expt.py` is designed to do this along with other setup stuff.   We could extend this script to support parallels using GDASApp.

Reply:
Replacing `try except` with 

`r2d2_config['experiment'] = config.get('experiment', 'oper_gdas')`
is a good suggestion.  

`experiment` is always defined in a parallel.   Thus, we always search for the requested file using the variable `experiment` in the path.   This is what we want for cycled files like `satbias` and `satbias_cov`.  We don't want this for (usually) fixed files like `tlapse`.   I can hardwire the `tlapse` `r2d2.fetch` to only look in `oper_gdas`.  

If the yaml used to run the stand-alone `run*py` scripts do not specify `experiment`, the default `oper_gdas` will be used.  I think this is the desired behavior.